resources/assets/css/tabler-adjustments.css

```diff
@@ -416,4 +416,8 @@ a.actions-buttons-column.show:after {
 [data-bs-theme=dark] a.actions-buttons-column:hover:after,
 [data-bs-theme=dark] a.actions-buttons-column.show:after {
     color: var(--tblr-dark-text);
+}
+
+body {
+    overflow-y: auto !important;
 }
```

Comment:
This is the only change not related to this PR (hide scroll bar when not needed).
It is from when I worked on only this branch. I figured out such a small change can be left here.

Reply:
I agree to have this scrollbar fix here... but from what I see this scrollbar fix introduces a new (small) problem. When at first there's no scrollbar, then the scrollbar appears, the whole page shifts a few pixels to the left on loading, so the user sees a small "page jump". Watch the "Backpack" text and the menu here:

![CleanShot 2023-04-19 at 10 46 03](https://user-images.githubusercontent.com/1032474/233005234-6a8b6f70-199c-4131-91e7-dc3f84a80896.gif)

Reply:
@tabacitu you are right, that looks ugly 😞

I wasn't able to find a clean solution, since showing/hiding the scrollbar seems to always carry that side-effects. That's because the scrollbar is not positioned on top of the content (it's not recommended either for obvious reasons).

We can use `overflow-y: scroll;` which will hide it when not needed it, but Tabler will still show the gap reserved for the scrollbar, which it doesn't look nice. The bright side? No jumping effect like the one you are showing on the gif.

<img width="588" alt="Screenshot 2023-04-19 at 11 17 19" src="https://user-images.githubusercontent.com/33960976/233013391-24be69fc-e3ee-451c-ae86-2446de55c054.png">

What do we do?

Reply:
![luke-skywalker-reaction-gif](https://user-images.githubusercontent.com/1032474/233020901-fe47a656-2e72-4954-8a32-803796296623.gif)

Seems like we have no good options 😔 What if we don't customize it at all?! I mean... what I would like, to be honest, is to have the MacOS behaviour. The scrollbar appears when you scroll.

I guess what we have here is better than the regular UX on Windows, where the scrollbar is probably ugly AF... but it doesn't really make sense on MacOS. So maybe... we leave it like Tabler did it on Windows (always visible, or at least the gutter)... and just don't customize anything on MacOS?!

We need some extra eyes on this with different operating systems. @promatik ? @pxpm ? What's the Windows perspective on this?